.github/CODEOWNERS

```diff
@@ -1,28 +1,24 @@
-/.github/               @terraform-google-modules/terraform-samples-git-admins
-/test/                  @terraform-google-modules/terraform-samples-git-admins @terraform-google-modules/cft-admins
-/build/                 @terraform-google-modules/terraform-samples-git-admins @terraform-google-modules/cft-admins
+/*                      @terraform-google-modules/terraform-samples-git-admins @terraform-google-modules/terraform-samples-reviewers @terraform-google-modules/cloud-samples-infra
+/.github/               @terraform-google-modules/terraform-samples-git-admins @terraform-google-modules/cloud-samples-infra
+/test/                  @terraform-google-modules/terraform-samples-git-admins @terraform-google-modules/cft-admins @terraform-google-modules/cloud-samples-infra
+/build/                 @terraform-google-modules/terraform-samples-git-admins @terraform-google-modules/cft-admins @terraform-google-modules/cloud-samples-infra
+
 /bigquery/              @terraform-google-modules/bigquery-terraform-swe @terraform-google-modules/terraform-samples-reviewers
-/cloud_scheduler/       @terraform-google-modules/terraform-samples-reviewers
 /cloud_sql/             @terraform-google-modules/infra-db-sdk @terraform-google-modules/terraform-samples-reviewers
 /cloudvpn/              @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /composer/              @terraform-google-modules/cloud-dpes-composer @terraform-google-modules/terraform-samples-reviewers
 /compute/               @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /dns/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
-/eventarc/              @terraform-google-modules/torus-dpe @terraform-google-modules/terraform-samples-reviewers
-/functions/             @terraform-google-modules/torus-dpe @terraform-google-modules/terraform-samples-reviewers
 /gke/                   @terraform-google-modules/eks-team @terraform-google-modules/terraform-samples-reviewers
 /lb/                    @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
-/looker/		@terraform-google-modules/cloud-looker-docs @terraform-google-modules/terraform-samples-reviewers
+/looker/		            @terraform-google-modules/cloud-looker-docs @terraform-google-modules/terraform-samples-reviewers
 /media_cdn/             @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /network_connectivity/  @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /privateca/             @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
-/run/                   @terraform-google-modules/torus-dpe @terraform-google-modules/terraform-samples-reviewers
```

Comment:
Why are these being removed?

Reply:
torus-dpe team is no longer in use and being removed from repositories. Once torus-dpe is removed from this row, the value of keeping this rule as an override seems missing. Is there a need to keep something in place?

Reply:
I see; I believe the intent here is that samples-reviews from the top level will catch this. 

I think this *isn't* entirely best practice, and not something we should be suggesting others do, but given this isn't one of our Big Four, I think it's fine? 

(Strictly speaking samples should never be added without expert reviewers, but the fact that in this specific case the expert reviewers moved teams to the superteam, I think it's fine)

Reply:
Clarifying, the big four have this, and the intent is that cloud-samples-infra is able to approve things like changes to README, Makefile, and CONTRIBUTING without having authority over the samples themselves. This PR should only have removed unused CODEOWNERS of samples, not moved authority around.